ice_in

```diff
@@ -1,450 +1,156 @@
-  tr_aero = .false.
-  tr_fsd = .false.
-  tr_fy = .true.
-  tr_iage = .true.
-  tr_iso = .false.
-  tr_lvl = .true.
-  tr_pond_lvl = .true.
-  tr_pond_topo = .false.
-  tr_snow = .false.
 /
 &thermo_nml
-  a_rapid_mode = 0.5e-03
-  aspect_rapid_mode = 1.0
-  conduct = "MU71"
-  dsdt_slow_mode = -1.5e-07
-  kitd = 1
-  ktherm = 2
-  phi_c_slow_mode = 0.05
-  phi_i_mushy = 0.85
-  rac_rapid_mode = 10
+  dsdt_slow_mode = -5e-08
 /
 &dynamics_nml
-  advection = "remap"
-  alphab = 20.0d0
-  arlx = 300.
-  brlx = 300.
-  cf = 17.0
-  coriolis = "latitude"
-  cstar = 20.0d0
-  e_plasticpot = 2.0
-  e_yieldcurve = 2.0
   k1 = 8.0d0
-  k2 = 15.0d0
-  kdyn = 1
-  krdg_partic = 1
-  krdg_redist = 1
-  kridge = 1
-  kstrength = 1
-  ktens = 0.0
-  ktransport = 1
-  mu_rdg = 4.0
-  ndte = 120
-  pstar = 2.75d4
-  revised_evp = .false.
-  seabed_stress = .false.
-  seabed_stress_method = "LKD"
-  ssh_stress = "coupled"
-  threshold_hw = 30.0d0
 /
 &shortwave_nml
-  ahmax = 0.3
-  albedo_type = "default"
-  albicei = 0.45
-  albicev = 0.75
-  albsnowi = 0.73
+  ahmax = 0.1
+  albicei = 0.44
+  albicev = 0.86
+  albsnowi = 0.7
   albsnowv = 0.98
-  dt_mlt = 1.50
+  dt_mlt = 1.0
   kalg = 0.0
-  r_ice = 0.0
-  r_pnd = 0.0
-  r_snw = 1.25
-  rsnw_mlt = 1500.
-  shortwave = "dEdd"
+  r_snw = 0.0
   sw_dtemp = 0.01d0
   sw_frac = 1.0d0
   sw_redist = .true.
 /
 &ponds_nml
   dpscale = 1.0e-3
-  frzpnd = "cesm"
-  hp1 = 0.01
-  hs0 = 0.0
-  hs1 = 0.03
-  pndaspect = 0.8
-  rfracmax = 0.85
-  rfracmin = 0.15
+  frzpnd = "hlid"
+  rfracmax = 1.0
 /
 &snow_nml
-  drhosdwind = 27.3
-  rhosmax = 450.0
-  rhosmin = 100.0
-  rhosnew = 100.0
-  rsnw_fall = 100.0
-  rsnw_tmax = 1500.0
-  snw_T_fname = "unknown"
-  snw_Tgrd_fname = "unknown"
-  snw_aging_table = "test"
-  snw_drdt0_fname = "unknown"
-  snw_filename = "unknown"
-  snw_kappa_fname = "unknown"
-  snw_rhos_fname = "unknown"
-  snw_tau_fname = "unknown"
-  snwgrain = .false.
-  snwlvlfac = 0.3
-  snwredist = "none"
-  use_smliq_pnd = .false.
-  windmin = 10.0
 /
 &forcing_nml
-  atmbndy = "default"
-  calc_strair = .true.
-  calc_tsfc = .true.
   emissivity = 0.95d0
-  fbot_xfer_type = "constant"
-  formdrag = .false.
   highfreq = .true.
-  ice_ref_salinity = 4.0d0
-  l_mpond_fresh = .false.
-  natmiter = 5
-  nfreq = 25
-  restart_coszen = .false.
-  rotate_wind = .true.
-  saltflux_option = "prognostic"
-  tfrz_option = "mushy"
+  tfrz_option = "linear_salt"
   update_ocn_f = .true.
   ustar_min = 0.005
-  wave_spec_file = "unknown_wave_spec_file"
-  wave_spec_type = "none"
 /
 &domain_nml
-  add_mpi_barriers = .false.
   block_size_x = 20
   block_size_y = 16
-  distribution_type = "sectrobin"
-  distribution_wght = "blockall"
-  ew_boundary_type = "cyclic"
   maskhalo_bound = .true.
   maskhalo_dyn = .true.
   maskhalo_remap = .true.
-  max_blocks = 8
+  max_blocks = 10
   ns_boundary_type = "tripole"
   nx_global = 360
   ny_global = 300
-  processor_shape = "square-ice"
+  processor_shape = "slenderX2"
```

Comment:
```suggestion
  processor_shape = "slenderX1"
```
to match access-om2

Reply:
With the block size and nx/ny settings here, we have 18 blocks in the x direction and 19 in the y direction.

With slenderX1, this gives 19 blocks per processor but only 18 processors would be used. With slenderX2 we get 9/10 blocks per processor and 36 processors used.

Now that I write this, maybe we should change the block_size_x to 16 and block_size_y to 15 with slenderX2 as this gives still 10 blocks per processor but they are slightly smaller and all 48 processors are actually used.

Reply:
ok it doesn't matter a lot at this stage as the core count will probably change